packages/zosfiles/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe z/OS files SDK package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Missing RECFM values results in the end user not able to create datasets accordingly. [#1639](https://github.com/zowe/zowe-cli/issues/1639)
```

Comment:
Maybe you can change this wording - we tend to say what's been done versus just stating the issue. So for example "Added missing RECFM values so that end user can create datasets accordingly." Not sure if this request is worth holding the PR back so will approve and leave that decision up to @t1m0thyj

Reply:
that's a good catch 😋 
I agree that it will help keep the changelog consistent.
I don't mind approving once Amber's comment is addressed 😋

Reply:
I made the suggested tweak, pls review and start workflows if okay. Thanks.